### PR TITLE
Apply fixes option

### PR DESCRIPTION
**Pull request by ellesmith88**
## Overview

This PR is for https://github.com/roocs/34e-mngmt/issues/68

Changes:

* Add apply fixes option

### Related Issue / Discussion

Following on from the call on Friday - I'm still not really sure what to do about the `daops` version. I've added the `apply_fixes` `daops` branch into `install_requires` but let me know what is best.

### CODE REVIEW

**Comment by agstephens** [comment]
@ellesmith88 Looks good to me :-)

File: setup.py [hunk 1]
```diff
@@ -50,6 +50,7 @@
       packages=find_packages(),
       include_package_data=True,
       install_requires=[
+          'daops @ git+https://github.com/roocs/daops.git@apply_fixes_option',
```
Comment (cehbrecht) [inline setup.py:53]: ... update daops first and use master branch before merge.

**Review by cehbrecht — APPROVED** [review]

**Comment by cehbrecht** [comment]
@ellesmith88 
> 
> Following on from the call on Friday - I'm still not really sure what to do about the `daops` version. I've added the `apply_fixes` `daops` branch into `install_requires` but let me know what is best.

I would also have done it like this :)

For local development I would install the dependent library (daops) manually from source into my conda environment.

**Comment by ellesmith88** [comment]
@cehbrecht This is failing because it's not using the most up to date `roocs.ini` from `roocs-utils` but I can't see where to fix it

**Comment by cehbrecht** [comment]
@ellesmith88 the `roocs.ini` for testing is generated using a template:
https://github.com/roocs/rook/blob/993cf2cc701ddf74caebcf0f7e9fef4c39c2df66/tests/common.py#L19

**Comment by ellesmith88** [comment]
> @ellesmith88 the `roocs.ini` for testing is generated using a template:
> https://github.com/roocs/rook/blob/993cf2cc701ddf74caebcf0f7e9fef4c39c2df66/tests/common.py#L19

Doesn't this only override the base directories for the tests? Some tests are failing because it can't find the` elasticsearch `section in the config, which is something I added yesterday and is only on the master branch of `roocs-utils`, not in a version.

**Comment by cehbrecht** [comment]
> > @ellesmith88 the `roocs.ini` for testing is generated using a template:
> > https://github.com/roocs/rook/blob/993cf2cc701ddf74caebcf0f7e9fef4c39c2df66/tests/common.py#L19
> 
> Doesn't this only override the base directories for the tests? Some tests are failing because it can't find the`elasticsearch`section in the config, which is something I added yesterday and is only on the master branch of `roocs-utils`, not in a version.

well ... it generates the complete `roocs.ini` used for testing. We need this to dynamically set the the root path of the test data. You can add the missing parts of the config to the template.

**Comment by ellesmith88** [comment]
> well ... it generates the complete `roocs.ini` used for testing. We need this to dynamically set the the root path of the test data. You can add the missing parts of the config to the template.

This will solve the problem but I have done the same thing in `daops` without needing to edit the template. I think its because when `clisops` is installed it installs `roocs-utils` 0.1.5 as a dependency and then the git version in `setup.py `doesn't override this, so the wrong `roocs-utils` is being used.

**Comment by cehbrecht** [comment]
> > well ... it generates the complete `roocs.ini` used for testing. We need this to dynamically set the the root path of the test data. You can add the missing parts of the config to the template.
> 
> This will solve the problem but I have done the same thing in `daops` without needing to edit the template. I think its because when `clisops` is installed it installs `roocs-utils` 0.1.5 as a dependency and then the git version in `setup.py `doesn't override this, so the wrong `roocs-utils` is being used.

ok ... so to get the right versions, `clisops` also needs to be installed from git master ...

**Comment by cehbrecht** [comment]
@ellesmith88 looks like it is working now :) Ok to merge?

**Pull request merged**